### PR TITLE
Create issue template for new tool support request

### PR DESCRIPTION
**Pull request by sumyapp**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new-tool-support-request.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
+
+**Additional context**
+Add any other context or screenshots about the feature request here.
+
+
+## Support [ANALYSIS_TOOL_Name]
```
Comment (ybiquitous) [inline .github/ISSUE_TEMPLATE/new-tool-support-request.md]: Typo?

```suggestion
## Support [ANALYSIS_TOOL_NAME]
```

**Comment by ybiquitous** [comment]
@sumyapp You can use `[skip ci]` within a commit message not to run slow CI. 😉

File: .github/ISSUE_TEMPLATE/new-tool-support-request.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+name: New tool support request
+about: Suggest new analysis tool support for this project
+title: Support [ANALYSIS_TOOL_NAME]
+labels: enhancement
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
Comment (yykamei) [inline .github/ISSUE_TEMPLATE/new-tool-support-request.md]: I think "e.g." is more common than "ex.".

File: .github/ISSUE_TEMPLATE/new-tool-support-request.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+A clear and concise description of any alternative solutions or features you've considered.
+
+**Additional context**
+Add any other context or screenshots about the feature request here.
+
+
+## Support [ANALYSIS_TOOL_Name]
+
+- Source code URL: [URL]
+- Official Site/Document: [URL]
```
Comment (yykamei) [inline .github/ISSUE_TEMPLATE/new-tool-support-request.md]: I'm not sure, but I believe "Documentation" is more common.

File: .github/ISSUE_TEMPLATE/new-tool-support-request.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  - https://help.sider.review/tools/cplusplus/cpplint
+- hadolint support
+  - https://github.com/sider/runners/issues/581
+  - https://github.com/sider/runners/pull/625
+  - https://github.com/sider/sider-docs/pull/282
+
+## TODO
+
+- [ ] Write smoke test(with some linter's Option)
+- [ ] Add pattern for success end, waring exit, error exit
```
Comment (yykamei) [inline .github/ISSUE_TEMPLATE/new-tool-support-request.md]: warning?

By the way, I might miss the point of this, but perhaps, writing smoke tests may include this requirement.

File: .github/ISSUE_TEMPLATE/new-tool-support-request.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+- [ ] Add config option
+- [ ] Add test for error exit
```
Comment (yykamei) [inline .github/ISSUE_TEMPLATE/new-tool-support-request.md]: How about this?

> (Optional) Add unit tests for the new linter's `Processor` if needed.

File: .github/ISSUE_TEMPLATE/new-tool-support-request.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+- [ ] Write document
```
Comment (yykamei) [inline .github/ISSUE_TEMPLATE/new-tool-support-request.md]: > Write documentation accordingly

File: .github/ISSUE_TEMPLATE/new-tool-support-request.md (hunk 5 above)
Comment (sumyapp) [reply .github/ISSUE_TEMPLATE/new-tool-support-request.md]: Sorry, I don't know... I copied this text from https://github.com/sider/runners/pull/625

File: .github/ISSUE_TEMPLATE/new-tool-support-request.md (hunk 2 above)
Comment (sumyapp) [reply .github/ISSUE_TEMPLATE/new-tool-support-request.md]: This is GitHub's template 🤣

File: .github/ISSUE_TEMPLATE/new-tool-support-request.md (hunk 4 above)
Comment (sumyapp) [reply .github/ISSUE_TEMPLATE/new-tool-support-request.md]: Okay. I combined this item and the first item.

**Pull request merged**